index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<title>portfolio</title>
+	<link href="styles.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body> 
+	
+	<nav>
+		<section>
+			<h1> Emily Clarkson</h1>
+			
+			<div class="page-nav">
+				<a class="about-link" href="#about">about</a>
+				<a class="work-link" href="#portfolio">work</a>
+				<a class="contact-link" href="#footer">contact</a>
+			</div>
+			
+			<div>
+				<a class="cv" href="#">download CV</a>
+			</div>
+				
+		</section>
+		
+	</nav>
+	
+	<header>
+		<section class="justify-content">
+  		   <div class="header-img">
+  				<img src= "file:///Users/emilyclarkson/Desktop/lettering-free/lettering-free/1080x1080/1_0037-new.png">
+  			</div>
+	    </section>
+	</header>
+	
+	<div id= "about">
+		
+		<section class="justify-content">
+			
+			<div class= "about-info">
+		       <h1>I'm Emily Clarkson</h1>
+			   <h2>Currently a student at IO Academy, based in London</h2>
+			   <p>Information blah blah blah. Information blah blah blah. Information blah blah blah. Information blah blah blah. Information blah blah blah. Information blah blah blah coding this. Information blah blah blah. Information blah blah blah. Information blah blah blah, coding that. </p>
+		   </div>
+		   <div>
+				<img src= "file:///Users/emilyclarkson/Desktop/w2/academy-aboutme-pic.png">
+			</div>
+				 
+		</section>
+	</div>
+	
+	<div class= "project-title">
+		
+		<section class="justify-content">
+		  <h1>see my latest work</h1>
+		  <p>Get an inside look into my creative process and see what's coming along</p>
+	</section>
+	
+	</div>
+	
+  <main id="portfolio">
+	  <div class="scrolling-wrapper">
+		
+	   <div class="project">
+			  <img src= "file:///Users/emilyclarkson/Desktop/w2/joshua-reddekopp-SyYmXSDnJ54-unsplash.jpg">
+			  <div class="project-info">
+				  <h3>project name</h3>
+				  <p>little intro to the project blah blah blah code is so exciting.</p>
+				  
+				  <button class="learn-more">learn more</button>
```

Comment:
You should use a tags instead of buttons here, you can't actually make a button link to a different page (atleast not without using javascript). They are designed to be used inside of form elements.

You can probably just change your button CSS to point to the a tags and it will likely still work (or will require some small tweaks)